addons/l10n_au/data/localisation.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+	<data noupdate="1">
+		<record id="base.AUD" model="res.currency">
+			<field name="position">before</field>
+			<field name="active" eval="True"/>
```

Comment:
yes when installing a CoA, we [reactive its currency](https://github.com/odoo/odoo/blob/a56c0ccbe4a1c9c64f80fd256fd2add24806f7f1/addons/account/models/chart_template.py#L710)